tests/unit/cpp_api.cpp

```diff
@@ -574,6 +574,22 @@ TYPED_TEST(CppApiTestFixture, TestNodeRemoveProperty) {
   ASSERT_EQ(node.Properties().size(), 0);
 }
 
+TYPED_TEST(CppApiTestFixture, TestRelationshipRemoveProperty) {
+  mgp_graph raw_graph = this->CreateGraph(memgraph::storage::View::NEW);
+  auto graph = mgp::Graph(&raw_graph);
+  auto node_1 = graph.CreateNode();
+  auto node_2 = graph.CreateNode();
+  auto relationship = graph.CreateRelationship(node_1, node_2, "RELATIONSHIP");
```

Comment:
All good, but usually we have for relationship first letter in uppercase, `:Relationship`